src/ui/base-page-storyEdit.js

```diff
@@ -18,6 +18,58 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         pageSetup: {
             hiddenEditorClass: "hidden"
         },
+        model: {
+            // the initial state is only the editor ui showing.
```

Comment:
Could you improve this comment to better explain 3 steps for editing a story and how the visibility of them are controlled. Thanks.